.claude/hooks/PreToolUse/invoke_false_completion_gate.py

```
@@ -0,0 +1,187 @@
+
+    # Skip if stdin is TTY
+    if sys.stdin.isatty():
+        return 0
+
+    # Read stdin JSON
+    try:
+        stdin_data = sys.stdin.read()
+        if not stdin_data.strip():
+            return 0
+        hook_input = json.loads(stdin_data)
+    except (json.JSONDecodeError, Exception):
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)